### PR TITLE
Fix/Change to one endpoint [cancel order]

### PR DESCRIPTION
**Pull request by ishiasaka**

### CODE REVIEW

**Review by Npwskp — DISMISSED** [review]
lgtm good job 🥇 🥵

File: src/utils/transformation.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { BadRequestException } from '@nestjs/common';
+import { TransformFnParams } from 'class-transformer';
+
+export class Transformation<T> {
+  constructor(
+    private readonly transformFunction: (value: any) => T,
+    private readonly nullable = true,
+    private readonly protoName = '',
```
Comment (Npwskp) [inline src/utils/transformation.ts:8]: why `protoName` is empty string, does it need to be type that u expect them to put in?

Comment (ishiasaka) [reply src/utils/transformation.ts:8]: protoName is a optional parameter to show expected type that dto want. Default value is empty string but you can take a name of the type to parameter.

Comment (Npwskp) [reply src/utils/transformation.ts:8]: gotcha

**Review by Npwskp — APPROVED** [review]
lgtm 🥵

**Review by lazykern — APPROVED** [review]
Just wanna tell you that I hate JS as much as I hate the college

**Pull request merged**